src/main/java/com/atlassian/jira/cloud/jenkins/common/client/JenkinsAppEventRequest.java

```diff
@@ -0,0 +1,69 @@
+package com.atlassian.jira.cloud.jenkins.common.client;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class JenkinsAppEventRequest extends JenkinsAppRequest {
```

Comment:
Introduced this class for "event" requests (i.e. builds and deployments events).